Tests/NIOTests/EventLoopFutureTest.swift

```diff
@@ -32,6 +32,197 @@ class EventLoopFutureTest : XCTestCase {
         let f = EventLoopFuture<Void>(eventLoop: eventLoop, error: EventLoopFutureTestError.example, file: #file, line: #line)
         XCTAssertTrue(f.isFulfilled)
     }
+    
+    func testFoldWithMultipleEventLoops() throws {
```

Comment:
@weissi Does this test suffice for multiple `eventLoop`s?

Reply:
yes, thanks!